_data/flux-commands.yaml

```diff
@@ -119,6 +121,41 @@ commands:
             command: flux alloc -N 2 --queue=foo
           - title: Create an allocation in the background with two nodes on queue 'foo'
             command: flux alloc --bg -N 2 --queue=foo
+
+  - name: flux bulksubmit
+    groups:
+      - name: Format String Examples
+        items:
+          - title: Pipe inputs into flux bulksubmit
+            command: seq 1 4 | flux bulksubmit echo {}
+          - title: Use GNU parallel to map the same inputs in
```

Comment:
It sounds like you're saying we're using gnu parallel, which I don't think we are.  I think you mean "gnu parallel style input" or something to that effect.

Reply:
Oh I thoughts that is what the three dots were - @grondo what are the dots from?

Reply:
Do you mean the triple-colon syntax `:::`? That is borrowed from GNU Parallel, but we don't use or depend on GNU parallel in any way.

Reply:
ahh gotcha, missed that part! I'll remove it.